test/test_prims.py

```diff
@@ -1094,6 +1094,46 @@ def test_reshape_view_method(self, device, dtype):
         result_refs = refs.view(a, *new_shape)
         self.assertEqual(result_eager, result_refs)
 
+    @dtypes(torch.float32)
+    def test_as_strided_scatter(self, device, dtype):
```

Comment:
instead of adding one-off tests, it would make sense to add a few reference inputs to `as_strided_scatter` reference inputs in OpInfos in `common_methods_invocation.py`, then e.g. this error would have been discovered earlier.

Reply:
Thanks, let me look into that